credentials/academic-credential/credential.json

```diff
@@ -0,0 +1,68 @@
+{
+    "@context": [
+        "https://www.w3.org/2018/credentials/v1",
+        "https://w3c-ccg.github.io/vc-ed/plugfest-1-2022/jff-vc-edu-plugfest-1-context.json",
```

Comment:
Do you really want to include this plugfest-1-context in addition to the OB context? I think you're going to get some term collisions, as this one was created only to backfill for a not-yet-complete OB context.

Reply:
Context files for PF3 updated now here: https://w3c-ccg.github.io/vc-ed/plugfest-3-2023/

Reply:
I'll update the PF context, drop the OB, and then check the expansion still works.
@SLisXL - I don't see a link to a PF3 context file on the page you linked and the example badges on the page don't include a PF context.
Can you help me see what I'm missing?

Reply:
@SLisXL that page still links to `https://purl.imsglobal.org/spec/ob/v3p0/context.json` which is the older context URL. It should now be (per Kerri) `https://purl.imsglobal.org/spec/ob/v3p0/context-3.0.2.json`. I sent in [a PR for the context files](https://github.com/w3c-ccg/vc-ed/pull/47) hosted in that repo...but missed the web page links. I'll send in another PR.

Reply:
@SLisXL here's the PR to update the homepage link: https://github.com/w3c-ccg/vc-ed/pull/50

Reply:
Sorry - thought we had updated that!

Reply:
No worries at all 👍